### ISSUE TITLE
As a law enforcement officer (LEO) or forest service employee, I can find out which permits are valid on any given day.

### ISSUE THREAD

**Issue by sadlerw** [nciinc/fs-intake-module#19]

**Comment by MelissaBraxton** [comment]
Revised based on LEO research.

**Comment by ASprinkle** [comment]
Primary use case for LEOs is to check and see if the user has a permit for the current season/valid permit(s). Secondary use case is for LEO to review the history of the user purchasing permits/expired permits (query range).  Searching by last name, first name (LEO) would likley be preferred, but report should populate an email address to help with same name conflicts.

**Comment by lauraGgit** [comment]
We might want to have a further discussion about this story would trigger a System of Record Notice (SORN).

**Comment by ASprinkle** [comment]
Yes you are correct - anytime we store information in a database we have to do a SORN. I am not clear on what the steps to accomplishing that are. Is it a simple notice to the public like something in the Federal Register?  Also - what are the requirements if we're not storing PII - or rather would we be since it would store a first and last name....?

**Comment by JaneZC** [comment]
Perhaps this would fall under an existing SORN? https://www.fs.fed.us/im/foia/pasystems.htm 

There is one for special uses ... https://www.fs.fed.us/im/foia/24.pdf

**Comment by ASprinkle** [comment]
I really do not know the answer to this.

**Comment by lauraGgit** [comment]
So my understanding is that up until 19 we did not have need to have a SORN because there was previously no way to search based on name. It is a little different then just storing PII in a DB. 

I don't think that we would be able to leverage the existing SORN because this the query is not for the special use permits. I recommend we flag for a discussion with FS.

**Comment by ASprinkle** [comment]
At a minimum - the ability to query permit numbers would be helpful.  Which could unblock this story.  Name query would be preferred.

**Comment by sadlerw** [comment]
Let's make this story for building a way to query a specific permit by id, and then make another for querying by permit holder name and that one can be cancelled if you decide you don't want to do the SORN?

**Comment by ASprinkle** [comment]
I have created story #327 to address the name query issue and possible need for a SORN.  This story, #19 would not require a name field in the query or the report.  This story, #19 is specific to being able to query by permit number.

**Comment by sadlerw** [comment]
@bryanmcfadden here's a possible layout with radio buttons:

![Screen Shot 2018-01-29 at 11.06.09 AM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/606a48e9-8356-48d3-9287-7c8098bf0aea)

![Screen Shot 2018-01-29 at 11.06.02 AM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/dbc6be67-523f-4b0d-8f82-addaa3b1d4f7)

**Comment by sadlerw** [comment]
![Screen Shot 2018-01-29 at 1.26.47 PM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/e5635cf2-97fc-46aa-be27-2c5f860c685d)

**Comment by bryanmcfadden** [comment]
@sadlerw - I think this design will work for now.

I think using the radio buttons in this case is ideal. The benefit over a select box is that you get to see both options that are available to you immediately (no clicking to view selections). If we get to 4 or more options, then we can reconsider the approach..but this should work for now.

I put in a little space on this version..(same thing as yours just aligned)

![image](https://user-images.githubusercontent.com/16689692/35531175-3e93af40-0504-11e8-9515-26de2ec9e58d.png)

**Comment by ASprinkle** [comment]
Can we change "id" to "ID number"?

**Comment by sadlerw** [comment]
![Screen Shot 2018-01-29 at 1.58.42 PM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/cdf2b0d0-9d0b-439c-a19c-58c1ae870ace)

**Comment by bryanmcfadden** [comment]
@sadlerw - I noticed there is no HR under the 'Get report' button. Can we add one in? This keeps separation between the user inputs and report. (also ..it keeps consistency with the rest of the site)

![image](https://user-images.githubusercontent.com/16689692/35531699-c2dd483c-0505-11e8-9b54-563a8356b20f.png)

**Comment by sadlerw** [comment]
added hr

**Comment by MelissaBraxton** [comment]
- Curious why the word "valid" is there. It implies that folks can't query expired permits.
 - The form field for permit ID is far too long. Make it closer to the number of characters that folks will actually enter. 
 - Minor thing - Can we decide whether its "create a report" or "generate a report"?

**Comment by cameronwolf** [comment]
Great points @MelissaBraxton. I have made those updates. What do you think of this?
<img width="691" alt="screen shot 2018-02-05 at 11 08 54 am" src="https://user-images.githubusercontent.com/7295948/35814803-06754cc0-0a65-11e8-9dd9-b4af7241088f.png">
